src/bin/help.ts

```diff
@@ -0,0 +1,230 @@
+import chalk from "chalk";
+
+import { logLine } from "../shared/cli/lines.js";
+
+export function logHelpText(version: string): void {
```

Comment:
> `version: string`

I suspect it might be better to figure out the version only if `logHelpText` is called. Otherwise the extra work of figuring that out will happen even if the user doesn't need it.

_(ignore this if you end up having the CLI always log the version - see later comment)_